NavigationTarget[] targets
bool ignore_angle

